app/src/models/status.ts

```diff
@@ -1,6 +1,4 @@
 import { DiffSelection, DiffSelectionType } from './diff'
-import { OcticonSymbol } from '../ui/octicons'
```

Comment:
The red flag for me about these functions was that we had models referencing types defined in the UI folder. With this change we've moved `iconForStatus` closer to the things it needed to import.